.github/workflows/release-release-artifacts.yml

```
@@ -315,11 +315,20 @@ jobs:
+        uses: ffurrer2/extract-release-notes@v2
+        with:
+          changelog_file: nym-vpn-android/CHANGELOG.md
+          # for prereleases, extract the changelog from the [unreleased] section
+          prerelease: ${{ inputs.github_release_type != 'release' }}
+
       - name: Add release notes (General)
         env:
           RELEASE_NOTES: ${{ env.RELEASE_NOTES }}
           VERSION: ${{ needs.resolve.outputs.version }}
           CORE_CHANGES: ${{ steps.core-changelog.outputs.release_notes }}
+          ANDROID_CHANGES: ${{ steps.a-changelog.outputs.release_notes }}
```

actionlint (1.7.12)

[error] property "a-changelog" is not defined in object type {android-changelog: {conclusion: string; outcome: string; outputs: {string => string}}; core-changelog: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)